assets/app/widgets/suse-lazy-spinner/suse-lazy-spinner.directive.js

```diff
@@ -0,0 +1,60 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('crowbarWidgets')
+        .directive('suseLazySpinner', suseLazySpinner)
+        .constant('SUSELS_DEFAULT_DELAY', 2000);
```

Comment:
How about
`.constant('SUSE_LAZY_SPINNER', {DEFAULT_DELAY: 2000});`?

I'm not fully convinced a directive file is the right place for a constant to be defined, but I don't have a strong suggestion neither